src/utils/date-helpers.js

```diff
@@ -26,7 +26,10 @@ function getTzShortName(tzAbbrevation) {
         if(tzAbbrevation.includes('GMT')) {
             return tzAbbrevation;
         }
-
+        
+        if(tzAbbrevation.includes('Coordinated Universal Time')){
```

Comment:
We can use a constant variable to indicate the string, also whether case sensitivity should be taken into consideration

Reply:
Done.